src/main/java/com/example/java/test/junior/developer/security/KeycloakConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package com.example.java.test.junior.developer.security;
+
+import lombok.Setter;
+import org.keycloak.adapters.springboot.KeycloakSpringBootProperties;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.admin.client.KeycloakBuilder;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+//TODO: Розібратись з ConfigurationProperties (як діставати дані). username ?
+@Setter
+@Configuration
+// не зовсім розібрався як штука нижче працює (автоматично підставляє через setter?), витягнув дані з keycloakProperties
+@ConfigurationProperties(prefix = "keycloak")
```

Comment:
Let's create a separate class `KeycloakProperties` and map our values from application.properties there.
Then you can autowire it here instead of KeycloakSpringBootProperties